components/TextField/react.js

```diff
@@ -67,6 +71,8 @@ const TextField = ({
         onChange={event => onChange(event.target.value, event)}
         onClick={onClick}
         value={fieldValue}
+        disabled={disabled}
+        readOnly={readonly}
```

Comment:
I'm torn. I see React uses readOnly, but HTML spec uses readonly. Actually this is fine, but made me do this emoji hard 🤔